src/pages/fallback/CommonFeedbackMessage.tsx

```diff
@@ -3,17 +3,34 @@ import { Trans } from 'react-i18next'
 
 import { GeyserFeedbackFromUrl } from '../../constants'
 
-export const CommonFeedbackMessage = () => {
+export const CommonFeedbackMessage = ({prefix}: {prefix: string | ''}) => {
   return (
-    <Text fontSize="20px">
-      <Trans i18nKey={'If the problem persists let us know via. <1>telegram</1> or this <3>feedback form.</3>'}>
-        {'If the problem persists let us know via. '}
-        <Link key="telegram" href="https://t.me/+EZ5otIPhVcxhMmFk" isExternal>
-          telegram
+    <Text variant="body1" textAlign="center">
+      {prefix}
+      <Trans
+        i18nKey={'If the issue persists let us know through our <1>feedback form</3> or reach out on <3>Telegram</1>'}
```

Comment:
This pattern wouldn't work for translations. Here's the suggested change:
```suggestion
        i18nKey={'If the issue persists let us know through our <1>feedback form</1> or reach out on <3>Telegram</3>'}
```

The way this works is, each block is part of an array, and if there is a different kind of block apart from normal text it goes to the next slot in the array. So in this example:
[
"'If the issue persists let us know through our".
" feedback form", (index is 1)
"or reach out on",
"Telegram" (index is 3)
}
More on this here: https://react.i18next.com/latest/trans-component#how-to-get-the-correct-translation-string